Jenkinsfile_k8s

```diff
@@ -198,6 +198,16 @@ pipeline {
               }
             }
           }
+          stage('Publish Docker image') {
+            steps {
+              if ((env.BRANCH_IS_PRIMARY) && (env.TAG_NAME)) {
+                infra.withDockerCredentials {
+                  def dockerImage = "jenkinsciinfra/jenkins-agent-${env.agent_type}:${env.TAG_NAME}"
+                  docker push dockerImage
+                }
+              }
+            }
```

Comment:
Keyword `if` is not supported in a declarative pipeline.
Either you wrap it under a `script` step, or you use a `when` block.

Reply:
```suggestion
            steps {
              script {
                if ((env.BRANCH_IS_PRIMARY) && (env.TAG_NAME)) {
                  infra.withDockerCredentials {
                    def dockerImage = "jenkinsciinfra/jenkins-agent-${env.agent_type}:${env.TAG_NAME}"
                    docker push dockerImage
                  }
                }
              }
            }
```